craft_application/commands/remote.py

```diff
@@ -28,17 +28,14 @@
 from craft_application.launchpad.models import Build, BuildState
 from craft_application.remote.utils import get_build_id
 
-OVERVIEW = """\
+OVERVIEW = """
```

Comment:
Fixes this:

![image](https://github.com/user-attachments/assets/f603391b-96c2-47b9-ac63-204d9f1c0286)